src/community/resources/contribution-day-organizer-handbook.md

```diff
@@ -0,0 +1,162 @@
+
+| Time      | Title                                                                             | Details                                                                                                                                                                                                                                                                                                                                                                          |
+|-----------|-----------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| 9AM-10AM  | Coffee and Introductions                                                          | Start the event off with an introduction round for the attendees, a description of the venue, Wi-Fi information, social media information (hashtags), communication mediums (i.e. Slack channel), highlight people to contact for help and key locations such as toilets, coffee and smoking areas. This helps to make sure people feel as comfortable as possible at the event. |
+| 10AM-11AM | How to contribute, find issues to work on and set up your development environment | An introduction on how to go about contributing, how to find issue to work on and how to get a local dev environment going. This is all about helping give attendees context which is especially important for first-time contributors. Unless your attendees are all veteran Magento contributors, consider having some amount of time allotted for this.                       |
+| 11AM-1PM  | Development Time                                                                  | Get hacking!                                                                                                                                                                                                                                                                                                                                                                     |
+| 1PM-2PM   | Lunch                                                                             | nom nom                                                                                                                                                                                                                                                                                                                                                                          |
```

Comment:
Choose a better phrase "nom" is slang and this should be avoided as English as a Second Language readers may not understand